changes/ee/feat-11447.en.md

```diff
@@ -0,0 +1 @@
+Added CLI command to wipe session and retained message data on the whole cluster.
```

Comment:
```suggestion
Added CLI command to wipe sessions and retained messages data on the whole cluster.
```

Reply:
I think "session" and "retained message" here are more like qualifiers/adjectives than nouns; "data" already encapsulates the plural? :thinking: